testcases/InstallUpstreamKernel.py

```diff
@@ -112,10 +126,10 @@ def is_url(path):
                     self.cv_HOST.copy_test_file_to_host(
                         self.config_path, dstdir=os.path.join(linux_path, ".config"))
             con.run_command("make %s" % self.config)
-            log.debug("Compile and install linux kernel")
-            con.run_command("make -j %d -s && make modules && make modules_install && make install" %
```

Comment:
Eh, keep the compile and build in the common path and move the case specific bits below. The `make modules` step can be dropped since the modules are built in the default target. The `modules_install` step should also be done with `-j %d` since it makes the module installation significantly faster.